app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::SessionsController < ApplicationController
+  def create
+    user = User.find_by(email: params[:email])
```

Comment:
initially was going to search with user id but email is unique so this works as intended